devices.js

```diff
@@ -12,6 +12,10 @@ var devices_recommended = {
   "Allnet": {
     "ALL0315N": "allnet-all0315n"
   },
+  
+    "Aruba": {
+    "AP-303": "aruba-ap-303"
```

Comment:
+ AP11 -> aruba-ap11

Reply:
I would go for "Instant On AP11" here. IMHO this puts the least confusion on the user.